policies/tagging/policy.json

```diff
@@ -87,15 +87,15 @@
                 },
                 {
                   "field": "tags['environment']",
-                  "notIn": "[parameters('allowedEnvironmentNames')]"
+                  "in": "[parameters('allowedEnvironmentNames')]"
```

Comment:
why have these been swapped from notIn to in this doesn't seem right? ?

Reply:
...reverting back to original logic and condition